src/components/FoodCard/index.tsx

```diff
@@ -24,12 +25,14 @@ export default function FoodCard({
   return (
     <article className={styles.foodCard}>
       <div className={styles.foodImageWrapper}>
-        <Image src={FoodImageSvg} alt={title} fill />
+        <Image src={foodImage} alt={title} fill />
       </div>
       <div className={styles.foodCardTextContent}>
         <div className={styles.foodCardTextWrapper}>
           <h3 className={styles.foodCardTitle}>{title}</h3>
-          <p className={styles.foodCardDescription}>{description}</p>
+          <p className={styles.foodCardDescription}>
+            {limitNumberOfCharacters(description)}
+          </p>
```

Comment:
```suggestion
          <p className={styles.foodCardDescription}>
            {limitNumberOfCharacters(description)}
          </p>
```
You can use the prop "truncate" to do this, it is better for SEO since the screen reader will still get the full text.

Reply:
@EricksonSiqueira Could you give me an example of use?